src/createPackage/models/createPackageManager.ts

```diff
@@ -338,6 +343,81 @@ export class CreatePackageManager {
     await fsPromise.writeFile(metadataFilePath, recordMetadata);
   }
 
+  public async createExportJsonMetadata(job: JobExportResponse): Promise<void> {
+    this.logger.info(`Creating metadata.json file for gpkg in path "${job.parameters.relativeDirectoryPath}" for jobId ${job.id}`);
+    const record = await this.rasterCatalogManager.findLayer(job.internalId as string);
+    const featuresRecords = parseFeatureCollection(job.parameters.roi);
```

Comment:
isn't it already exists in the job parameters? why do you need to parse it again?

Reply:
not all resolution params, and it occured on triggering and not polling